metals/src/main/scala/scala/meta/internal/metals/MetalsEnrichments.scala

```diff
@@ -508,4 +508,8 @@ object MetalsEnrichments
       trailingTokens.find(predicate)
   }
 
+  implicit class XtensionMetalsCommand(command: Command) {
```

Comment:
This method can be defined on `Command` directly, we mostly use extension methods for 3rd party types

Reply:
It could be that we have a few `toLSP` extension methods on mtags types from the time mtags didn't depend on lsp4j.